README.md

```diff
@@ -570,6 +570,51 @@ Displays a leaderboard sorted by funds raised (highest first) for a single speci
 </html>
 ```
 
+#### MMF Leaderboard (Campaign)
```

Comment:
Can we rename this to Fitness Leaderboard ?

In Supporter we have Fitness Activities and do not mention MMF anywhere except on the Manage Applications page for connecting to MMF.  We will be adding other fitness providers in the future so it won't just be MapMyFitness.